### PR TITLE
[8.0][ADD][survey_one_choice_per_column] Restrict answers (one answer per column) on a matrix

### PR DESCRIPTION
**Pull request by tarteo**

### CODE REVIEW

File: survey_one_choice_per_column/README.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+============================
+Survey One Choice Per Column
+============================
+
+This module adds an option 'One Choice Per Column' to the matrix type of the matrix question type.
+
+Usage
+=====
+
+* go to Marketing -> Surveys -> Surveys;
```
Comment (elicoidal) [inline survey_one_choice_per_column/README.rst]: @tarteo Thanks for the contribution
Please use enumerated list ('#.') to describe procedure

File: survey_one_choice_per_column/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2016 ONESTEiN BV (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Survey One Choice Per Column',
+    'images': [],
```
Comment (elicoidal) [inline survey_one_choice_per_column/__openerp__.py]: remove empty keys

File: survey_one_choice_per_column/views/assets.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 ONESTEiN BV (<http://www.onestein.eu>)
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<openerp>
+    <data>
+        <template id="survey_one_choice_per_column_assets" inherit_id="website.assets_frontend" name="Survey One Choice Per Column Assets">
```
Comment (elicoidal) [inline survey_one_choice_per_column/views/assets.xml]: Can you improve a the format with shorter lines?
It helps the review in general and gets better readability

**Comment by tarteo** [comment]
@elicoidal Thank you for your feedback.

**Comment by astirpe** [comment]
:+1:

**Comment by ehahouimohssine** [comment]
:+1: Tested the functionality

**Comment by rafaelbn** [comment]
Hi thans a lot for this:

As commented with @hbrunn please add test to you code

"untested coded needs fixing in my book, and the conventions seem to agree: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#tests"

Thanks. I will test it when coverage it solved.

**Comment by rafaelbn** [comment]
Hi @tarteo now travis fails

**Comment by tarteo** [comment]
@rafaelbn I'm still working on the unit tests, I will clean the code when I'm finished.

**Comment by rafaelbn** [comment]
OK! Thanks! :smiley:

**Comment by tarteo** [comment]
@rafaelbn I finished the unit tests.

**Comment by rafaelbn** [comment]
Thanks a lot @tarteo functionally :+1: 
Could you please review coverage? It is still red.

**Pull request merged**